src/c/types/array.c

```diff
@@ -24,16 +24,25 @@
 //                INTERNAL SERIALIZATION IMPLEMENTATION
 // -------------------------------------------------------------------
 
+void ucdr_serialize_continuous_memory(ucdrBuffer* ub, const uint8_t* array, const uint32_t size)
```

Comment:
This function can have a define to compile with the previous version in order to improve the performance.

Reply:
Outdated